charts/tesk/README.md

```diff
@@ -0,0 +1,77 @@
+# Deployment instructions for TESK
+
+Helm chart and config to deploy the TESK service. Tested with Helm v3.0.0.
+
+## Usage
+
+First you must create a namespace in Kubernetes in which to deploy TESK. The
+command below creates all deployments in the context of this namespace. How
+the namespace is created depends on the cluster, so it is not documented here.
+
+To deploy the application:
+ * modify [`values.yaml`](values.yaml)
+ * Create a `secrets.yaml` file. You need to fill up the `username` and `password` of the ftp server configured in WES. If you activated authentication (auth.mode == 'auth' in `values.yaml`), you need as well the `client_id` and `client_secret`, otherwise the auth section must be removed:
```

Comment:
If you want to set TESK as an OIDC client to get user tokens, turn on its authentication (auth.mode == 'auth' in `values.yaml`). Please supply OIDC `client_id` and `client_secret` to this. (PLEASE NOTE: If you have another OIDC client for fetching user's OIDC token & don't want to use TESK as client, remove/comment `auth` section)

Reply:
`auth.mode` activated both the OIDC client and the requirement to provide a token to interact to TESK. Or in other workds, if `auth.mode` is set to `noauth`, then TESK has anonymous access.

I will write a mash up of your suggestions and what I just explained.